index.bs

```diff
@@ -303,6 +284,61 @@ Loader instances are initially created with the internal slots described in the
 
 <h3 id="registry-abstract-operations">Abstract Operations for Registry Objects</h3>
 
+<h4 id="registry-GetCurrentStage" aoid="GetCurrentStage">GetCurrentStage(entry)</h4>
+
+The abstract operation GetCurrentStage with argument entry performs the following steps:
+
+<emu-alg>
+1. If Type(_entry_) is not Object, throw a *TypeError* exception.
+1. Let _stages_ be _entry_.[[Pipeline]].
+1. Return the first element of _stages_.
```

Comment:
it can't be empty. it is created by EnsureRegistered() or .install(), either way you can't never make it empty, you can only upgrade to an existent stage.